src/utils/json.rs

```diff
@@ -0,0 +1,9 @@
+use serde::{Deserialize, Deserializer};
+
+pub fn try_deserialize<'de, D, T>(deserializer: D) -> Result<Option<T>, D::Error>
```

Comment:
This is automagically applies for all options? Or when is this registered to be used by serde?

Reply:
Ahh I see serialize_with ;)